api/queries_repo.go

```diff
@@ -1391,3 +1407,51 @@ func RepoExists(client *Client, repo ghrepo.Interface) (bool, error) {
 		return false, ghAPI.HandleHTTPError(resp)
 	}
 }
+
+// ListLicenseTemplates fetches available repository license templates.
+// It uses API v3 because license template isn't supported by GraphQL.
+func ListLicenseTemplates(httpClient *http.Client, hostname string) ([]License, error) {
+	var licenseTemplates []License
+	client := NewClientFromHTTP(httpClient)
+	err := client.REST(hostname, "GET", "licenses", nil, &licenseTemplates)
+	if err != nil {
+		return nil, err
+	}
+	return licenseTemplates, nil
+}
+
+// LicenseTemplate fetches an available repository license template.
+// It uses API v3 because license template isn't supported by GraphQL.
+func LicenseTemplate(httpClient *http.Client, hostname string, licenseTemplateName string) (License, error) {
+	var licenseTemplates License
+	client := NewClientFromHTTP(httpClient)
+	err := client.REST(hostname, "GET", fmt.Sprintf("licenses/%v", licenseTemplateName), nil, &licenseTemplates)
```

Comment:
Minor nit for keeping this in line with [similar logic](https://github.com/search?q=repo%3Acli%2Fcli%20%22client.REST%22&type=code), avoid complex single statements:

```suggestion
	path := fmt.Sprintf("licenses/%s", licenseTemplateName)
	err := client.REST(hostname, "GET", path, nil, &licenseTemplates)
```